frontend/packages/kubevirt-plugin/src/components/virtualization-overview/RunningVmsPerTemplateCard.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import { ChartDonut, ChartLabel, ChartLegend } from '@patternfly/react-charts';
+import { Card, CardBody, CardHeader, CardTitle } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import { VirtualMachineModel } from '../../models';
+import { kubevirtReferenceForModel } from '../../models/kubevirtReferenceForModel';
+import { VMKind } from '../../types';
+
+const getVmsPerTemplateMap = (vms, vmsLoaded) => {
+  let numVMs = 0;
+  const map = {};
+  vmsLoaded &&
+    vms.forEach((vm) => {
+      const template = vm?.metadata?.labels['vm.kubevirt.io/template'];
```

Comment:
nitpick: please use this const `LABEL_USED_TEMPLATE_NAME` instead

Reply:
Done!